src/MassTransit.Abstractions/Configuration/Middleware/IReceiveEndpointConfigurator.cs

```diff
@@ -84,6 +84,6 @@ void ConfigureMessageTopology<T>(bool enabled = true)
         /// Add the observable receive endpoint as a dependency
         /// </summary>
         /// <param name="connector"></param>
-        void AddDependency(IReceiveEndpointObserverConnector connector);
+        void AddDependency(IReceiveEndpointDependencyConnector connector);
```

Comment:
So, given the naming of dependency (something I need) vs dependent (something that needs me):

AddDependency(dependent);